### PR TITLE
Add CentOS + Fedora Support

### PR DESCRIPTION
**Pull request by con5cience**

### CODE REVIEW

**Comment by achaussier** [comment]
Hello, thanks for your work.

I'll check it tomorrow or this week-end.

**Comment by con5cience** [comment]
I've been using this daily for months and everything works fine. :) Not sure what's up with the tests.

**Comment by achaussier** [comment]
Hello,

I've set develop branch to be the main target of PR and apply some changes to fix testing and update the role to be validated using Molecule.

I move the target of the PR to develop branch and check the change

File: tasks/configure.yml [hunk 1]
```diff
@@ -2,11 +2,13 @@
 
 # Manage Collectd configuration
 
+
+
 - name: 'CONFIG | MAIN | Manage collectd main configuration file'
   become: True
   template:
     src: 'etc/collectd/collectd.conf.j2'
```
Comment (achaussier) [inline tasks/configure.yml:10]: Now, I prefix all roles paths by {{ role_path }}, Can you update this statement please

File: tasks/configure.yml [hunk 2]
```diff
@@ -2,11 +2,13 @@
...
-    dest: '/etc/collectd/collectd.conf'
+    dest: '/{{ collectd_config_directory }}/collectd.conf'
```
Comment (achaussier) [inline tasks/configure.yml:11]: > collectd_config_directory

 value must be the absolute path, same as other variables

**Review by achaussier — CHANGES_REQUESTED** [review]
Hello,

- can you rebase from develop branch please
- can you fix the following issues
- can you add a molecule test platform to test the new features ?
- can you update Readme and meta files please

Thanks

**Pull request closed without merging**